db/migrate/20211109111813_create_playlists.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePlaylists < ActiveRecord::Migration[6.1]
+  def change
+    create_table :playlists do |t|
+      t.string :name
```

Comment:
```suggestion
      t.string :name, null: false
```

De manière général, si tu es sûre qu'un champ doit être présent, il faut ajouter cette contrainte d'intégrité dans la base de données, comme ça on est SUR que ce champ sera toujours rempli